configurations.gradle

```diff
@@ -1,10 +1,8 @@
 configurations {
-    provided
-    compile.extendsFrom provided
```

Comment:
`implementation` replaced `compile` and `provided didn't seem to be used in any special way, so I replaced all instances of `provided` with `implementation`. Let me know if I was off base here.